src/Faker/Provider/PhoneNumber.php

```diff
@@ -8,6 +8,40 @@ class PhoneNumber extends Base
 {
     protected static $formats = ['###-###-###'];
 
+    /**
+     * @see https://github.com/giggsey/libphonenumber-for-php/blob/master/src/CountryCodeToRegionCodeMapForTesting.php
+     */
+    protected static $e164Formats = [
```

Comment:
Is there a reason why the calling codes for de_AT and de_CH are missing while de_DE and fr_FR included?

Reply:
The default array is only copied from the linked source above.
There are even more countries missing but I think that this list is big enough for testing and every country provider can define the specific ones.